pyproject.toml

```diff
@@ -141,7 +141,7 @@ exclude = [
 [tool.slotscheck]
 exclude-modules = '''
 (
-  ^ddtrace.(contrib|vendor)
+  ^ddtrace.(contrib|vendor).*
```

Comment:
Was this not working before then? This is now excluding all submodules, but it used to NOT exclude them right?

Reply:
Honestly I forgot why I made this change. I'll revert it to see what fails

Reply:
Revert changed nothing. Thanks for calling this out